opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
```

Comment:
this is how you are supposed to do it with msvc. this is a separate compilation unit, the define (and the *PREPROCESSOR* definitions it enables) is only around while compiling this .cpp file so i don't see what the fuss is about.